### ISSUE TITLE
/tlsissuer: unable to get certificate issuer (Invalid argument)

### ISSUE THREAD

**Issue by alfredh** [baresip/baresip#1357]
tested with commit 2987b4e4198b70ba75d8cad0d90e806914055e57 on OSX 10.14
OpenSSL 1.1.1k 

```
/tlsissuer
unable to get certificate issuer (Invalid argument)
```

**Comment by cspiel1** [comment]
Will have a look on this tomorrow.

**Comment by juha-h** [comment]
If I remember correctly, it printed certificate during call, but not
that of the server.

**Comment by juha-h** [comment]
I checked that managed to get these:
```
/tlsissuer
TLS Cert Issuer:  CN=CAcert Class 3 Root,OU=http://www.CAcert.org,O=CAcert Inc.
/tlssubject
TLS Cert Subject:  CN=test.tutpro.com
```
But what is the point, because the same info can be got using openssl command?  It would be much more useful to get tls info of the server.

**Comment by cspiel1** [comment]
You are right. It prints the information of the certificate configured with `sip_certificate`. Right, a TLS info of the peer certificate during a call would be useful. We could
- add new commands to print this
- or rewrite the two commands. I have to check if, and how we are using already these commands. Most likely no one else uses them so far, because they where added not long ago.

**Comment by alfredh** [comment]
in my case, the sip_certificate option was empty.

my feedback is, that in such a case it should print a more userfriendly error-message
instead of EINVAL ... :)

**Comment by alfredh** [comment]
fixed in 2983f38df084c02c41af11922fc089932512f3f0
